src/cdk/explorer/tree/treeInspector.ts

```diff
@@ -63,3 +63,18 @@ export function getDisplayLabel(construct: ConstructTreeEntity): string {
 
     return construct.id
 }
+
+/** 
+* @param construct CDK construct
+*/
+export function isStateMachine(construct: ConstructTreeEntity): boolean {
```

Comment:
It might make more sense for this function to live directly in `constructNode.ts`, as right now that's the only place we'll be using it.